src/shell/hooks/useKeyboardShortcut.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from "react";
+import { useSelector } from "react-redux";
+
+/*
+Usage Example
+  useKeyboardShortcut("letter", () => callback());
+
+*/
+
+export function useKeyboardShortcut(key, callback) {
```

Comment:
Lets rename this to `useMetaKey`. 

The context of whether its a shortcut or not is irrelevant to the hook it's self.